core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowExecutionsResource.scala

```diff
@@ -20,37 +19,40 @@ object WorkflowExecutionsResource {
   final private lazy val context = SqlServer.createDSLContext()
 
   case class WorkflowExecutionEntry(
-      eId: UInteger,
-      vId: UInteger,
-      startingTime: Timestamp,
-      completionTime: Timestamp,
-      status: Byte,
-      result: String
-  )
+                                     eId: UInteger,
+                                     vId: UInteger,
+                                     startingTime: Timestamp,
+                                     completionTime: Timestamp,
+                                     status: Byte,
+                                     result: String,
+                                     bookmarked: Boolean // FIXME: how come we have this class at all, what's the point of jooq if we have this? it's redundant, no?
```

Comment:
As discussed this will be fixed in a future pr once we add the results info. Please add a todo and fix the comment wording

Reply:
Done.